http/vertx/src/test/java/io/quarkus/ts/vertx/AbstractVertxIT.java

```diff
@@ -41,6 +44,20 @@ public void httpServerParsing() {
         requests().get("/hello?name=you").then().statusCode(HttpStatus.SC_OK).body("content", is("Hello, you!"));
     }
 
+    @Test
+    public void httpClient() {
+        HttpClient httpClient = Vertx.vertx().createHttpClient();
+        httpClient.request(HttpMethod.GET, 1101, "localhost", "/hello")
```

Comment:
Hard-coded localhost and port is not good, find a way how to get these from the code, consult the framework wiki